ha/k8s_setup/ha_setup.py

```diff
@@ -261,7 +261,10 @@ def process(self):
             # TODO: Verify whether these newly added config is avilable in the confstore or not
             with open(const.HA_CONFIG_FILE, 'w+') as conf_file:
                 yaml.dump(conf_file_dict, conf_file, default_flow_style=False)
-            self._confstore = ConfigManager.get_confstore()
+            # Note: kv_enable_batch is setting True to commit all the kv keys in 1 time for time optimization.
+            # if kv_enable_batch set to true then commit operation needs to run to put all the key values in store.
+            kv_enable_batch_put = True
+            self._confstore = ConfigManager.get_confstore(kv_enable_batch=kv_enable_batch_put)
```

Comment:
yes, it is used like protected, but it's an existing class member I have not added newly.
it is used in 3 classes.

I am thinking not to change in this PR as these will be cleanup/correction changes, unnecessary it will increase the code chnages of this PR, but yes need to change.

Reply:
will fix in: https://jts.seagate.com/browse/CORTX-31319